Workstream Policy.md

```diff
@@ -103,6 +103,15 @@ The [Steering Group] approves new [Workstreams][Workstream] and determines the [
 [Contributions] are deemed to have been made jointly by both the individual submitting the
 [Contribution][Contributions] and any [Entity] identified in the [Contributor and Workstream Participant Agreement].
 
+##### Trivial contributions
+
+Trivial changes may be accepted without coverage from the agreement. A trivial change is one that fits one of the following categories:
+
+* Makes no changes to the visible text content (for example, fixing broken links, style sheets or markup), OR
+* Does not affect conformance (e.g. corrects a typographic error in a descriptive sentence), which should not change architectural or interoperability requirements for an implementation. Such changes should be small in scope (e.g., on the order of a few dozen characters changed at most); correcting non-normative code examples where the code clearly conflicts with normative requirements is fine, adding new non-normative code examples is not.
```

Comment:
> Such changes should be small in scope (e.g., on the order of a few dozen characters changed at most)

Is this consistent with PRs where we would want to mark them as trivial contributions, especially the number of characters?

Reply:
I think so, at least quite a few of them. Also, if it's repetitive I think the editor can use discretion to only count the non-repetitive instances.